pkg/cloud/vsphere/assets/cloud-controller-manager-deployment.yaml

```diff
@@ -111,6 +111,10 @@ spec:
             items:
               - key: cloud.conf
                 path: cloud.conf
+        - name: secret-vccm
+          secret:
+            secretName: {{ .globalCredsSecretName }}
+            optional: true
```

Comment:
What is vccm?

Reply:
vsphere cloud controller manager

Reply:
What's the motivation behind adding this new secret, I don't see quite how that's related to the goal of this PR? Is it that the vSphere CCM normally looks up the creds but we need a signal to the logic that this secret is being used?

Reply:
https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/pkg/cloud/vsphere/assets/cloud-controller-manager-deployment.yaml#L63

ccm reads this secret, and need to reroll deployment if credentials changed. This is a sorta crutch to achieve this.